frontend/src/components/VirtualList/Config.ts

```diff
@@ -31,7 +31,11 @@ export function hasHealth(r: RenderResource): r is SortResource {
 }
 
 export const hasMissingSidecar = (r: SortResource): boolean => {
-  return !isIstioNamespace(r.namespace) && !r.istioSidecar && !isGateway(r.labels);
+  return !isIstioNamespace(r.namespace) && !r.istioSidecar && !isGateway(r.labels) && !isWaypoint(r.labels);
+};
+
+export const hasMissingAmbient = (r: SortResource): boolean => {
```

Comment:
what does "missing ambient" mean?

Reply:
The first idea was to have a similar meaning that `missingSidecar`. But the implementation is not the same, so probably can be more meaningful with something like `noAmbientLabels`